rubocop-naming.yml

```
@@ -9,3 +9,7 @@ Naming/VariableNumber:
 # https://docs.rubocop.org/rubocop/cops_naming.html#naminginclusivelanguage
 Naming/InclusiveLanguage:
   Enabled: false
+
+# https://docs.rubocop.org/rubocop/cops_naming.html#namingblockforwarding
+Naming/BlockForwarding:
+  Enabled: explicit
```

@jmjurado23 I think this is not being correctly configured. explicit should be used with `EnforcedStyle`, and enabling this rule in a separate line. 

https://docs.rubocop.org/rubocop/cops_naming.html#namingblockforwarding

```suggestion
  Enabled: true
  EnforcedStyle: explicit
```